src/main/java/edu/kis/powp/factory/ShapeFactory.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.factory;
+
+import edu.kis.powp.command.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+
+public class ShapeFactory {
+
+
+    public static DriverCommand getShape(DriverManager driverManager, String type) {
+        switch (type) {
+            case "rectangle":
```

Comment:
Action should not depend on string value. Enum or separate listeners would be better. At least you should consider constants.